bcolz/carray_ext.pyx

```diff
@@ -1216,9 +1229,17 @@ cdef class carray:
             self.chunks.append(chunk_)
             cbytes += chunk_.cbytes
         self.leftover = leftover = nbytes % self._chunksize
+
```

Comment:
I think these changes break it? The tests all pass without it, so not sure why it would be needed.